### ISSUE TITLE
Configurable cameraview init pose

### ISSUE THREAD

**Issue by osrf-migration** [osrf/gzweb#66]
**Original report ([archived issue](https://osrf-migration.github.io/gzweb-gh-pages/#!/osrf/gzweb/issues/66)) by Isaac Saito (Bitbucket: [130s](https://bitbucket.org/%7B7ebec088-7c5d-41ea-83c4-ea79199d7afd%7D/)).**

----------------------------------------
In `.world` file we can define the cameraview's init pose like this, which `Gzweb` 1.2.0 seems to be ignoring for me.

```
     </scene>
+    <gui>
+     <camera name="camera_world_main">
+       <pose>57 15 1 0 0.05 1.4</pose>
+     </camera>
+    </gui>
   </world>
```

Manually chaging `gzweb/http/client/gz3d.js` like below works.

```
  // camera
  this.camera = new THREE.PerspectiveCamera(
      60, window.innerWidth / window.innerHeight, 0.1, 1000 );
- this.defaultCameraPosition = new THREE.Vector3(0, -5, 5);
+ this.defaultCameraPosition = new THREE.Vector3(57, 15, 1);
  this.resetView();
```
```
GZ3D.Scene.prototype.resetView = function()
{
  this.camera.position.copy(this.defaultCameraPosition);
  this.camera.up = new THREE.Vector3(0, 0, 1);
- this.camera.lookAt(new THREE.Vector3( 0, 0, 0 ));
+ this.camera.lookAt(new THREE.Vector3( 62, 22, 0 ));
  this.camera.updateMatrix();
};
```

Still, this `resetView` isn't effective when the web page is reloaded. Pressing `Reset View` button on Gzweb works.

So in summary, issues I'm seeing:

1) `camera` tag in SDF is ignored.
2) `resetView` in `gzweb/http/client/gz3d.js` does not work when the Gzweb web page is loaded.
